details/operator-sdk/api/v1/astraconnector_types.go

```diff
@@ -22,6 +22,17 @@ type Astra struct {
 	Unregister        bool   `json:"unregister,omitempty"`
 }
 
+// AutoSupport defines how the customer interacts with NetApp ActiveIQ.
+type AutoSupport struct {
+	// Enrolled determines if you want to send anonymous data to NetApp for support purposes.
+	// +kubebuilder:default:=true
+	Enrolled bool `json:"enrolled"`
+
+	// URL determines where the anonymous data will be sent
+	// +kubebuilder:default:="https://216.240.31.151/put/AsupPut-setenv"
```

Comment:
https://216.240.31.151/put/AsupPut-setenv was a bad test example.  Would update this to the real staging url:
https://stagesupport.netapp.com/put/AsupPut